resources/functionMap.php

```diff
@@ -286,7 +286,7 @@
 'array_map' => ['array', 'callback'=>'?callable', 'array'=>'array', '...args='=>'array'],
 'array_merge' => ['array', 'arr1'=>'array', '...args='=>'array'],
 'array_merge_recursive' => ['array', 'arr1'=>'array', '...args='=>'array'],
-'array_multisort' => ['bool', '&rw_array1'=>'array', 'array1_sort_order='=>'array|int', 'array1_sort_flags='=>'array|int', '...args='=>'array|int'],
+'array_multisort' => ['bool', 'rw_array1'=>'array', 'array1_sort_order='=>'array|int', 'array1_sort_flags='=>'array|int', '...args='=>'array|int'],
```

Comment:
The name of the parameter is `array1`, not `rw_array1`.

Reply:
I saw many other param with a wrong name, like
- array_pop => rw_stack
- array_push => rw_stack
- array_shift => rw_stack
- array_splice => rw_input
- array_unsplice => rw_stack
- ...

There is at least 63 occurences of `&rw_` param which might need to be checked...

Do you want me to open another PR with such param renaming ?

Reply:
These aren't part of the name when there's `&` in front of them. There's an explanation at the top of functionMap.php